tools/panaroo/panaroo.xml

```diff
@@ -0,0 +1,156 @@
+        </section>
+        <!--Matching-->
+        <section name="matching" title="Matching"  expanded="false">
+            <param argument="--threshold" type="float" optional="true" value="0.98" label="Initial Clustering Threshold" help="Set the threshold for gene clustering. Default is 0.98."/>
+            <param argument="--family_threshold" type="float" optional="true" value="0.7" label="Family Sequence Identity Level" help="Set the level at which Panaroo collapses genes into putative families. Default is 0.7."/>
+            <param argument="--len_dif_percent" type="float" optional="true" value="0.98" label="Length Difference Percent" help="Set the length difference percent for initial gene clustering. Default is 0.98."/>
+            <param name="merge_paralogs" type="boolean" optional="true" truevalue="--merge_paralogs" falsevalue="" label="Merge Paralogs" help="Merging paralogs can be enabled. Disabled by default."/>
+        </section>
+        <!--Refind-->
+        <section name="refind" title="Refind"  expanded="false">
+            <param argument="--search_radius" type="integer" optional="true" value="5000" label="Refinding Search Radius" help="Set radius for the search in nucleotides for refinding genes. Default is 5000."/>
+            <param argument="--refind_prop_match" type="float" optional="true" value="0.5" label="Refinding Proportion Match" help="Set the proportion of the missing gene to consider as a match for refinding. Default is 0.5."/>
+            <param argument="--refind-mode" type="select" optional="true" label="Refind Mode" help="Set the stringency mode at which to re-find genes.">
+                <option value="default" selected="True">Default</option>
+                <option value="strict">Strict</option>
+                <option value="off ">Off</option>
+            </param>
+        </section>
+        <!--GRAPH CORRECTION -->
+
+        <!--Gene Alignment-->
+        <section name="gene_alignment" title="Gene Alignment"  expanded="false">
+            <param name="alignment" type="select" optional="true" label="Alignment" help="Run MSA either for the core genome or for every gene cluster.">
+                <option value="core" selected="True">Core genome alignment</option>
+                <option value="pan">Pan-genome alignment</option>
+            </param>
+            <param argument="--aligner" type="select" optional="true" label="Alignment method" help="Choose an aligner from either PRANK, Clustal or the default, MAFFT.">
+                <option value="mafft" selected="True">MAFFT</option>
+                <option value="clustal">Clustal</option>
+                <option value="prank">PRANK</option>
+                <option value="none">None</option>
+            </param>
```

Comment:
Add following parameters as well:

  --codons              Generate codon alignments by aligning sequences at the protein level
  --core_threshold CORE
                        Core-genome sample threshold (default=0.95)
  --core_subset SUBSET  Randomly subset the core genome to these many genes (default=all)
  --core_entropy_filter HC_THRESHOLD
                        Manually set the Block Mapping and Gathering with Entropy (BMGE) filter. Can be between 0.0 and 1.0. By default this is set using
                        the Tukey outlier method.